### PR TITLE
python3Packages.aqualogic: init at 2.3

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python library to interface with Hayward/Goldline AquaLogic/ProLogic pool controllers

https://github.com/swilson/aqualogic

This is a Home Assistant dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113000` at e348105a run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/2d19fe7d00557b29420c02373d4e2608">python37Packages.aqualogic</a></li></li>
    <li><a href="https://gist.github.com/9eba1528fc01e8b076e1baf7f76164fb">python38Packages.aqualogic</a></li></li>
    <li><a href="https://gist.github.com/b98891ced4fd3b4dec3bd79342bc53df">python39Packages.aqualogic</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `pythonOlder`.
Near pkgs/development/python-modules/aqualogic/default.nix:6:3:
```
  |
6 | , pythonOlder
  |   ^
```
</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113000` at e348105a run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/30aee0df7f5021d34f84cb397425ee72">python37Packages.aqualogic</a></li></li>
    <li><a href="https://gist.github.com/0c439015b85b7f6e47211c5c0f3a0a21">python38Packages.aqualogic</a></li></li>
    <li><a href="https://gist.github.com/eb24fcbfcd860ecdbc9a1c2251090bb7">python39Packages.aqualogic</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `pythonOlder`.
Near pkgs/development/python-modules/aqualogic/default.nix:6:3:
```
  |
6 | , pythonOlder
  |   ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/aqualogic/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyserial
+, pytestCheckHook
+, pythonOlder
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/aqualogic/default.nix:6]:
```suggestion
```

Comment (fabaff) [reply pkgs/development/python-modules/aqualogic/default.nix]: Indeed, that's not needed, thanks. Removed.

**Pull request merged**